packages/web/src/scss/components/Tabs/_tools.scss

```diff
@@ -0,0 +1,17 @@
+@use '../../tools/breakpoint';
+
+@mixin item-visibility($breakpoints) {
```

Comment:
We already have this code—as utility classes. Did you consider using them instead?

Reply:
True. Removed.